target/linux/ipq806x/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,15 @@ nbg6817)
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "$board:amber:wifi5" "phy0radio"
 	ucidef_set_led_netdev "wan" "WAN" "$board:white:internet" "eth1"
 	;;
+vr2600v)
+	ucidef_set_led_default "dsl" "DSL" "${board}:white:dsl" "0"
+	ucidef_set_led_usbport "usb" "USB" "${board}:white:usb" "usb4-port1" "usb2-port1"
```

Comment:
As far as I understand this covers only usb 2.0 devices attached to the usb port. Have a look at one of my earlier comments. I suggested some more usb ports to add and linked to comment where I got the informations from.